build

```diff
@@ -3,29 +3,40 @@
 # Stop on first error
 set -e
 
-print_help() {
+if [[ $1 == "-h" ]] || [[ $1 == "--help" ]]; then
 	echo "-h, --help	Print this message"
 	echo "-d, --debug	Build Lightspark with debug symbols"
 	echo "By default, this script does the release build of Lightspark."
+	exit 0
+fi
+
+check_cmake() {
+	if ! type cmake > /dev/null 2>&1; then
+		echo "cmake not found" >&2
+		exit 1
+	fi
 }
 
 case "$1" in
-	-h | --help)
-		print_help
-		exit 0
-		;;
 	-d | --debug)
 		mkdir -p obj-debug
 		cd obj-debug
+		check_cmake
```

Comment:
How about introducing check_cmds() ?

check_cmds() checks the availability of the commands needed for building the target like cmake and make.
Running check_cmds before making a directory may be better. If one doesn't install cmake, making a directory is a waste of CPU time. If one doesn't install make, running cmake is also a waste of CPU time.

Reply:
Good idea.